release.wdl

```diff
@@ -0,0 +1,238 @@
+#Thoughts--
+#Still need to manually check that the jars are being created in broad artifactory
+#Need to manually investigate if tests aren't being written
+#Need to manualy update the release portion of Cromwell
+#Need to make sure that the changelog.md file doesn't get weirdly altered when merging to master for Cromwell
+
+
+task basicRelease {
+   Array[String] dependency
+   String repo
+   String releaseV
+   String baseDir
+
+   command {
+     cd ${baseDir}/${repo}
+     git checkout develop
+     git pull
+
+     currentV=`grep "git.baseVersion " build.sbt | cut -d "=" -f 2 | tr -d \"\,`
+     echo "Version of ${repo} on the develop branch is $currentV"
+
+     if [ ${releaseV} -ne $version ]; then
+        echo "Need to update the ${repo} version on the develop branch to ${releaseV}" >&2
+        sed -i -e 's/'"$currentV"'/'"${releaseV}"'/' build.sbt
+        git add build.sbt
+        git commit -m "Update ${repo} version from $currentV to ${releaseV}"
+        if [ ${repo} -eq "wdl4s" ]; then #updating docs
+           sbt 'set scalacOptions in (Compile, doc) := List("-skip-packages", "better")' doc
+           git checkout gh-pages
+           mv target/scala-2.11/api ${releaseV}
+           git add ${releaseV}
+           git commit -m "API Docs"
+           #git push origin gh-pages
+
+           git checkout develop
+           sed -i -e 's/'"$currentV"'/'"${releaseV}"'/' README.md
+           git add README.md
+           git commit -m "Update ${repo} API docs references"
+        fi
+        #git push origin develop
+     fi
+
+    #git checkout master
+    #git merge --squash origin/develop
+    #git tag ${releaseV}
+    #git push origin master
+    #git push --tags
+   }
+
+   output{
+     String updatedVersion = releaseV
+   }
+}
+
+task wdltoolRelease {
+String baseDir
+String repo
+String releaseV
+String wdl4sV
+
+  command {
+    cd ${baseDir}/${repo}
+        git checkout develop
+        git pull
+
+        currentV=`grep "git.baseVersion " build.sbt | cut -d "=" -f2 | tr -d \"\,`
```

Comment:
Not sure if it's wdl friendly, but backticks are deprecated (here and other cases below). If it's compatible, use something like:

```bash
currentV=$(grep "git.baseVersion " build.sbt | cut -d "=" -f2 | tr -d \"\,)